script/foreman-rake

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 USERNAME=$(/usr/bin/id -un)
 RAKE_CMD=/usr/bin/rake
+BUNDLE=bundle
 BUNDLER_CMD=""
 
 die() {
   echo $*; exit 1
 }
 
 cd ~foreman || die "Cannot change to foreman home directory"
-[ -f Gemfile ] && BUNDLER_CMD="bundle exec"
+[ -h /usr/bin/foreman-ruby ] && BUNDLE="/usr/bin/foreman-ruby /usr/bin/bundle"
```

Comment:
Could this stay in packaging? e.g. set either BUNDLER_CMD or BUNDLE to this from the build script?